apps/registry/pages/api/[payload].js

```diff
@@ -14,6 +14,10 @@ const { Client } = require('pg');
 
 const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 
+if (!GITHUB_TOKEN && process.env.NODE_ENV !== 'development') {
+  throw new Error('GITHUB_TOKEN is not set in environment variables');
```

Comment:
When I did not specify the `GITHUB_TOKEN` in the environment, I would get an error message, that the github username does not exist, which made it harder to debug. This route will get issues, if we do not set a `GITHUB_TOKEN` because, as far as I understand, we will hit rate limits with the github api. This is why I think it is a good idea to fail hard and gracefully here instead of failing weirdly later